rob-shayne/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Lab 4</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+    <link rel="stylesheet" href="styles/icons.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+    <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+
+    <!--
+      TODOne: Add a script tag to hold the Handlebars template.
+        - Be sure to give it a meaningful id attribute.
+        - Move your old template structure to here. Remember that this template
+          needs to have the same structure as the old one!
+        - Insert placeholder keys.
+        - If your object doesn't have values you need yet, you can add those properties in the model.
+    -->
+    <script id="post-template" type="text/x-handlebars-template">
+      <article class="template" data-category="{{ category }}">
```

Comment:
you're correctly targeting data-category right here, you need to also do the same for data-author in order to keep both the filters working after adding handlebars.